frontend/projects/valtimo/components/assets/css/compatibility.scss

```
@@ -182,6 +182,11 @@ valtimo-modal {
 }
 
 .case-detail-overflow {
+  .cds--btn--primary .cds--btn__icon,
+  .cds--btn--primary .cds--btn__icon path {
+    fill: currentColor !important;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)